src/pretalx/orga/templates/orga/settings/room_list.html

```diff
@@ -4,7 +4,9 @@
 {% block settings_content %}
 
 <div>
-    <legend>{{ request.event.rooms.count }} {% trans "Rooms" %}</legend>
+    <legend>{{ request.event.rooms.count }}
+        {% blocktrans count count=request.event.rooms.count %}Room{% plural %}Rooms{% endblocktrans %}
```

Comment:
having this all in one line isn't particularly nice, but if I split it up into like so:
```html
    <legend>{{ request.event.rooms.count }}
        {% blocktrans count count=request.event.rooms.count %}
        Room
        {% plural %}
        Rooms
        {% endblocktrans %}
    </legend>
```
all the whitespace ends up in the .po file. :-/